_content/news/_posts/2018-11-12-clinic.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Coaches Clinic"
+date: 2018-11-12 20:00:00
+description: Op zaterdag 22 december organiseren we 2 coach clinics voor jeugdcoaches. 
+cover: /news/img/clinic-22-12-2018.jpg
+permalink: /news/2018-11-12-clinic/
+---
+
+Op zaterdag 22 december organiseren we in samenwerking met coach Gerrit Driessens, de headcoach van Kangoeroes Willebroek en coach van het jaar 2017, 2 coaches clinics voor jeugdcoaches.
+
+### Wat is een coaches clinic?
+
+Voor leken is de term 'clinic' een beetje misleidend. Het is niet dat er een bepaald probleem 'genezen' wordt in een clinic. Nee, een clinic kan je best vergelijken met een traditionele bedrijfspresentatie.
+
+In een clinic komt een ervaren coach uitleggen hoe hij/zij een bepaald aspect van het basketbal spel aanleert aan een team. Dit is vaak een combinatie van theoretische uitleg samen met voorbeeld oefeningen die gedemonstreerd worden door een selectie spelers.
+
+Een mooi voorbeeld van een clinic werd onlangs gegeven door coach Philip Wolk (de coach van onze Dames A) op de FIBA U14 Get Together. Waar hij uitlegt hoe hij de dribble hand-off in een motion offense aanleert.
+
+<iframe width="666" height="375" src="https://www.youtube.com/embed/rwbNeHWRzrE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Programma
+
+Op zaterdag 22 december staan er 2 clinics op het programma, beide gegeven door Gerrit Driessens.
+
+Van 9u30 tot 10u45 komt coach Driessens uitleggen koe we de trainingstijd zo goed mogelijk kunnen maximaliseren bij grote trainingsgroepen (gericht op U8 & U10) en hoe we de fysieke geletterdheid verder kunnen ontwikkelen aan de hand van spelvormen.
+ 
+En van 11u tot 12u15 komt hij uitleggen hoe we bij de latere leeftijden, U14-U16-U18/U19, kunnen zorgen voor progressie van de dribbel hand-off naar pick and roll. Hoe we dit kunnen aanleren en verder ontwikkelen.
+
+### Inschrijven
+
+Coaches kunnen zich inschrijven voor deze clinics via de website van basketbal vlaanderen.
+
+[http://basketbal.vlaanderen/agenda/detail/coach-clinic-gerrit-driessens-lummen](http://basketbal.vlaanderen/agenda/detail/coach-clinic-gerrit-driessens-lummen)
+
+De clinic gaat door in Sporthal De Vijfspron te lummen en is gratis voor alle coaches van Basket Lummen, maar is ook opgengesteld aan coaches uit andere clubs.
```

Comment:
De clinic gaat door in sporthal De Vijfsprong  in Lummen, coaches uit andere clubs zijn welkom. Voor coaches van Basket Lummen is de clinic gratis.